src/main/webapp/resources/cart.js

```diff
@@ -0,0 +1,65 @@
+$(document).ready(function () {
+    const lineItemTemplate = Handlebars.compile($("#line-item-template").html());
+    const paymentMethodTemplate = Handlebars.compile($("#dropdown-template").html());
+
+    const keyStorage = "my-cart";
+    const cart = localStorage.getItem(keyStorage);
+
+    if (cart) {
+        $("#cart-empty-badge").hide();
+
+    } else {
+        $("#product-table").hide();
+        $("#additional-info").hide();
+    }
+
+    let totalPrice = 0;
+    let rowNumber = 0;
+    let lineItems = $.map(JSON.parse(cart), function (value, index) {
+        const lineItemHtml = lineItemTemplate({
+            rowNumber: ++rowNumber,
+            productName: value.product.name,
+            count: value.count,
+            price: value.product.price * value.count
+        });
+
+        $("#cart-items").append(lineItemHtml);
+
+        totalPrice += value.product.price * value.count;
+        return {
+            productId: index,
+            count: value.count
+        };
+    });
+    $("#total-price").append(`<span>$</span>${totalPrice}`);
+
+    $.get("/app/payment-methods", function (paymentMethods, status, jqXHR) {
```

Comment:
The context part `/app` has to be extracted and defined globally since the context can be easily changed when redeploying.

Reply:
The context `/app` has to be defined within a separate file e.g. `context.js`. The `context.js` has to be included in all the templates which access REST API.